azure/postgres.sh

```diff
@@ -69,7 +69,7 @@ fn azure_postgres_set_max_size(instance, size) {
 # azure_postgres_set_version sets the Postgres version of a server.
 # `instance` is the postgres server instance.
 # `version` is the postgres server version.
-# Default: 9.5
+# Default: latest
```

Comment:
Defaulting to a "latest" is never a great idea for me, you won't have any kind of reproducibility, but if it is Azure defaults and we can't do anything against it no problem